api/python/provisioner/commands/__init__.py

```diff
@@ -1381,11 +1370,14 @@ def _passwordless_ssh():
         command = getattr(_mod, cmd_cls)
     except AttributeError:
         try:
-            cmd_mod = importlib.import_module(
-                f'provisioner.commands.{cmd_name}'
-            )
+            if "cluster_id" in cmd_name:
```

Comment:
no, please don't add any specific logic here. options:
- either keep get/set commands for cluster-id separated
- introduce support of new optional values inside `type: [module].Type` so module can be parsed from here and used instead of default value which is command name , e.g
```
get_cluster_id:
  type: cluster_id.GetClusterId
```
- implement support for scanning of the whole `commands` directory (advanced task, don't think it should be here)


I believe the 2nd option is the best choice for the moment.

Reply:
@andkononykhin Sure, thanks for the suggestion. Added support to handle types which are of `module.Type` format. In the hope that, as we migrate common/utility modules, this feature can be leveraged for those modules as well.